themes/default/layouts/index.html

```diff
@@ -271,29 +267,47 @@ <h6>{{ $item.title }}</h6>
         </div>
     </section>
 
-    <!-- Benefits -->
-    <section class="container mx-auto my-12 lg:my-24 relative">
-        <h2 class="text-center">{{ .Params.benefits.title }}</h2>
-        <div class="benefits-section relative">
-            <div class="shape-background home-benefits-section">
-                <div class="shape-container">
-                    <div class="circle-left"></div>
-                    <div class="circle-right"></div>
+    <section class="container mx-auto my-16 text-center px-3">
+        <div class="card bg-violet-200 p-6 rounded-2xl">
+            <h2 class="rainbow-text">Doing more with less</h2>
+            <p class="text-lg max-w-3xl mx-auto">
+                You can use Pulumi's Automation API to create tooling that help you and your engineers manage <span class="font-bold">10x the amount of resources</span> versus
+                traditional tooling.
+            </p>
+
+            <div class="flex flex-wrap text-left">
+                <div class="w-full lg:w-1/3 p-6">
+                    <p>
+                        What if IaC was a library and not just a CLI? Pulumi Automation API is the industry's first and only fully programmable infrastructure as code technology.
+                    </p>
+                    <p>
+                        Automation API use IaC as a library within a program to take direct infrastructure actions (create, update, destroy). This code can execute anywhere from a
```

Comment:
This feels like an important paragraph that could benefit from a bit more concreteness and a lighter tone. Take it or leave it, but I'd maybe suggest something more like this:   

```suggestion
                        With Automation API, you can embed Pulumi right into your application code to drive complex deployment workflows programmatically. Build command-line tools, desktop applications, self-service infrastructure portals, and more, all in your language of choice &mdash; without having to "shell out" to another CLI.
```